src/app/shared/components/Field/PlayerCard/index.tsx

```diff
@@ -48,8 +51,10 @@ const PlayerCard = (props: PlayerCardProps) => {
       >
         <div className="flex align-middle text-center">
           <PlayerIcon
+            isFaded={!isPlayerSelected}
```

Comment:
I don't love isFaded but agree that unSelected needs to be improved... 

hmmmm

Could just go with selected or isPlayerSelected